messaging/src/androidTest/java/io/lantern/messaging/MessagingTest.kt

```diff
@@ -453,7 +453,22 @@ class MessagingTest : BaseMessagingTest() {
                                     )
                                 )
                             )
-                            val imageAttachment = dog.createAttachment(assetToFile("image.jpg"))
+                            val imageAttachment = dog.createAttachment(
+                                assetToFile("image.jpg")
+                            )
+                            val audioAttachment = dog.createAttachment(
```

Comment:
I added an audio attachment to `testAttachments` to make sure that everything we need makes it into the attachment model.